zcash_client_sqlite/src/wallet/init/migrations/add_transaction_views.rs

```diff
@@ -437,7 +438,7 @@ mod tests {
 
         let usk = UnifiedSpendingKey::from_seed(&network, &[0u8; 32][..], AccountId::ZERO).unwrap();
         let ufvk = usk.to_unified_full_viewing_key();
-        let (ua, _) = ufvk.default_address();
+        let (ua, _) = ufvk.default_address(UnifiedAddressRequest::unsafe_new(false, true, true));
```

Comment:
Since migrations are correct as of a particular point in time, here we manually construct the request given the state of the codebase at the time that the migration was created.